docs/store/selectors.md

```diff
@@ -104,6 +104,24 @@ export const selectFeatureCount = createSelector(
 );
 ```
 
+There is also an overload on the `createFeatureSelector` method, which allows a better typechecking.
```

Comment:
Instead of adding this as an additional section, let's just make this the default in the example above.

Reply:
Yea I wasn't sure about this, but you just confirmed my thinking.
Should I also update the example app?

Reply:
Yes